### PR TITLE
Selectable cells

### PR DESCRIPTION
**Pull request by arsimonini**
Added the necessary components for selectable cells to be started as well as textComments

### CODE REVIEW

File: pythonfile.py [hunk 1]
```diff
@@ -128,13 +152,27 @@ def main():
         if stp_button.draw(win):
             print("Next Step is...")
 
+        # draws comments
+        title.draw(win, "Sue Dooku: The Sudoku Solver")
```
Comment (CierraO) [inline pythonfile.py]: 🤨

Comment (arsimonini) [reply pythonfile.py]: why not homeslice

Comment (CierraO) [inline pythonfile.py]:
```suggestion
        title.draw(win, "The Sudoku Solver")
```

**Review by CierraO — APPROVED** [review]
I'll approve it but you're on thin ice. Suspicious lack of comments. Do better

**Pull request merged**